arangod/Agency/ResignLeadership.cpp

```diff
@@ -426,7 +429,7 @@ bool ResignLeadership::scheduleMoveShards(std::shared_ptr<Builder>& trx) {
 
           MoveShard(_snapshot, _agent, _jobId + "-" + std::to_string(sub++),
                     _jobId, database.first, collptr.first, shard.first, _server,
-                    toServer, isLeader, true)
+                    toServer, isLeader, true, _undoMoves)
```

Comment:
Adding this here to make it more clear to readers of the code.
```suggestion
                    toServer, isLeader, /*remainsFollower*/ true, _undoMoves)
```